docs/src/content/docs/ko/guides/components.mdx

```diff
@@ -418,6 +418,36 @@ import { Steps } from '@astrojs/starlight/components';
 
 </Steps>
 
+### 배지
+
+import { Badge } from '@astrojs/starlight/components';
+
+상태나 라벨과 같은 작은 정보를 표시하려면 `<Badge>` 컴포넌트를 사용하세요.
+
+표시하려는 콘텐츠를 `<Badge>` 컴포넌트의 `text` 속성에 전달하세요.
+
+기본적으로 배지는 사이트의 테마 강조 색상을 사용합니다. 내장 배지 색상을 사용하려면 `variant` 속성의 값을 `note` (파란색), `tip` (보라색), `danger` (빨간색), `caution` (주황색), 'success'(녹색) 중 하나로 설정하세요. .
```

Comment:
Just to double check, is the ". ." at the end expected or should it be "." only?

Reply:
Thanks for the good point. It has been modified.